client/components/proposal/SupportTable.tsx

```diff
@@ -0,0 +1,45 @@
+import { FunctionComponent } from "react";
+import { Address } from "components/Address";
+import TokenAmount from "components/TokenAmount";
+import { ethers } from "ethers";
+
+interface SupportTableProps {
+  voters: Array<object>;
+}
+
+const SupportTable: FunctionComponent<SupportTableProps> = ({ voters }) => {
+  return (
+    <table className="table table-compact w-full">
+      <thead>
+        <tr>
+          <th className="pl-0">Address</th>
+          <th>Votes</th>
+        </tr>
+      </thead>
+      <tbody>
+        {voters.map((voter) => (
+          <tr key={voter.address}>
+            <td className="pl-0">
+              <Address address={voter.address} />
+            </td>
+            <td>
+              <TokenAmount
+                amount={ethers.utils.formatUnits(
+                  ethers.BigNumber.from(voter.votes)
```

Comment:
nit: BigNumber.from can throw if voter.votes isn't BigNumberish

Reply:
Updated component to catch!